### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by eabdalmufid** [SuspiciousLookingOwl/saweria-api#8]
![Screenshot_2024-07-24-21-30-15-540_com server auditor ssh client-edit](https://github.com/user-attachments/assets/f2425643-6aaa-434b-885e-bdf3fadcd613)

**Comment by SuspiciousLookingOwl** [comment]
Can you provide code example and version of the lib that you are using?
This code works fine for me on `2.0.1`
```ts
const client = new Client();

client.on("login", (user) => {
	console.log(user);
});

client.login("...", "...");
```

**Comment by eabdalmufid** [comment]
My code 
```javascript
import { Client } from "saweria";

const client = new Client();

client.on("login", (user) => {
	console.log("Logged in as: ", user.username);
});

client.login("...", "...");
```
![Screenshot_2024-07-27-21-36-26-808_com server auditor ssh client-edit](https://github.com/user-attachments/assets/fe9fade5-039c-475f-bf42-2f181c92e104)

**Comment by KuntilBogel** [comment]
Maybe your ip?

**Comment by eabdalmufid** [comment]
> Maybe your ip?

![IMG-20240728-WA0130](https://github.com/user-attachments/assets/72db48b7-415e-41f4-8ae6-545f2bd353be)
Yes, cloudflare 🥲
Is there a solution?

**Comment by SuspiciousLookingOwl** [comment]
tl;dr: no easy solution 😔

Read more here: https://github.com/SuspiciousLookingOwl/saweria-api/issues/4

**Comment by KuntilBogel** [comment]
looks like it bypassed cf if your ip is from asia

**Comment by eabdalmufid** [comment]
> looks like it bypassed cf if your ip is from asia

I use a US ip

**Comment by KuntilBogel** [comment]
That may cause the issue, I suggest using a proxy
